osvr/RenderKit/osvr_display_configuration.cpp

```diff
@@ -532,11 +532,6 @@ void OSVRDisplayConfiguration::parse(const std::string& display_description) {
     {
         auto const& distortion = hmd["distortion"];
 
-        /// Find out how many desired triangles, if this is specified.
-        /// If not specified, set to a default.
-        m_distortionDesiredTriangleCount =
-          distortion.get("desired_triangle_count", 200 * 64).asInt();
-
```

Comment:
Looks like another accidental revert of another pull request. Please merge master (or rebase).